quilt3_local/quilt3_local/api.py

```diff
@@ -0,0 +1,51 @@
+import ariadne.asgi
+import boto3
+import fastapi
+from botocore.exceptions import ClientError
+
+from .graphql import schema as graphql_schema
+
+
+class ApiException(Exception):
+    """
+    Base class for API exceptions.
+    """
+    def __init__(self, status_code, message):
+        super().__init__()
+        self.status_code = status_code
+        self.message = message
+
+
+sts_client = boto3.client("sts")
+session_cred = boto3._get_default_session().get_credentials()
+
+
+api = fastapi.FastAPI()
+
+
+@api.get("/api/auth/get_credentials")
+def get_credentials():
+    """
+    Obtains credentials corresponding to your role.
```

Comment:
```suggestion
    Obtain credentials corresponding to your role.
```
per [PEP 257](https://www.python.org/dev/peps/pep-0257/):

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".